resources/conf/config.toml.template

```diff
@@ -492,6 +492,10 @@ enabled = true
   # Enforcer related configurations
   [analytics.enforcer]
     [analytics.enforcer.configProperties]
+      # Enables elk analytics logging
+      type = "elk"
+      # Overrides default analytics publisher reporter class
+      "publisher.reporter.class" = "org.wso2.am.analytics.publisher.sample.reporter.CustomReporter"
       # Authentication endpoint for Choreo Analytics Publishing
       authURL = "$env{analytics_authURL}"
```

Comment:
Let's mention in the comment that, if type is ELK we do not need this property. 
Same goes with token.